### ISSUE TITLE
[Feature] multiple zendesk instances

### ISSUE THREAD

**Issue by CaseyLeab** [fivetran/dbt_zendesk#124]
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Describe the Feature

We have multiple zendesk instances in our organization, how would I setup this package so that I can create the models for each of the zendesk instances we have. 
(They are all in separate schemas; zendesk_concierge, zendesk, zendesk_partner_success, etc....)

### Are you interested in contributing this feature?

- [ ] Yes.
- [X] Yes, but I will need assistance and will schedule time during your [office hours](https://calendly.com/fivetran-solutions-team/fivetran-solutions-team-office-hours) for guidance.
- [ ] No.

**Comment by fivetran-jamie** [comment]
hey there! we were actually heavily considering rolling this out in the coming quarter. 

i am curious about your use case -- does each zendesk instance belong to a different department or subsidiary in your org?

**Comment by CaseyLeab** [comment]
> hey there! we were actually heavily considering rolling this out in the coming quarter.
> 
> i am curious about your use case -- does each zendesk instance belong to a different department or subsidiary in your org?

Yes, we have 5 different zendesk instances and each belongs to a different organization in the building and different metrics, tickets, and tracking associated to it. In zendesk they're separated by different subdomains so there is an individual connector for each in Fivetran.

**Comment by fivetran-jamie** [comment]
Gotcha! so the solution we were thinking of rolling out would union together these different instances' source data and produce 1 set of output models -- which you could split out into individual ones based on the `source_relation` column. would that work for your use case?

**Comment by CaseyLeab** [comment]
I think that would work. Thank you. Please keep me posted when you prioritize and plan to roll this out as our depts are bummed that ticket_metrics is the biggest table they aren't able to see at the moment.

**Comment by mariahjrogers** [comment]
Hi, I was wondering if you could provide an update on the progress of this issue @fivetran-jamie ? I am eager for this to be released!

**Comment by fivetran-jamie** [comment]
Apologies for not updating this ticket! This feature is available to use -- see instructions [here](https://github.com/fivetran/dbt_zendesk?tab=readme-ov-file#option-b-union-multiple-connections)
